api_v3/services/BaseEntryService.php

```diff
@@ -84,9 +88,27 @@ function addAction(KalturaBaseEntry $entry, $type = -1)
     	myNotificationMgr::createNotification(kNotificationJobData::NOTIFICATION_TYPE_ENTRY_ADD, $dbEntry, $dbEntry->getPartnerId(), null, null, null, $dbEntry->getId());
     	
 	    $entry->fromObject($dbEntry, $this->getResponseProfile());
+		$this->handleMultiLanguageMapping($entry);
 	    return $entry;
     }
 	
+	protected function handleMultiLanguageMapping(KalturaBaseEntry &$entry)
+	{
+		$defaultLanguage = $this->getPartner()->getDefaultLanguage();
```

Comment:
we wont have default language, in case request language was not specified we will return the db value